pages/components/textUtilities/textUtil.css

```diff
@@ -47,15 +47,15 @@
   font-size: 1em;
 }
 ul > .ul-item-solid:hover {
-  background-color: transparent;
+  background-color: var(--pale-white);
```

Comment:
I think you should keep your background color as transparent because if someone else is using your classes in their project and you have given a color here then it will create problems in their ui

Reply:
ok